sktime/distances/_edr.py

```diff
@@ -112,19 +212,17 @@ def _edr_cost_matrix(
     bounding_matrix: np.ndarray,
     epsilon: float,
 ):
-    """Compute the edr cost matrix between two time series.
+    """Compute the edr cost matrix between two timeseries.
```

Comment:
time series not timeseries ... or if you really dont like time series, use time-series, but there is no word timeseries